src/dataServices/dynamoDbDataService.test.ts

```diff
@@ -338,6 +338,32 @@ describe('UPDATE', () => {
         expect(serviceResponse.message).toEqual('Resource created');
         expect(serviceResponse.resource).toStrictEqual(expectedResource);
     });
+
+    test('ERROR: Id supplied for Update as Create is not valid', async () => {
+        // BUILD
+        const id = 'uuid:$deadbeef';

```

Comment:
Got a good laugh here :)

Reply:
That's a nod to my roots as an embedded developer!